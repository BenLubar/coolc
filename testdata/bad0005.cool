class Main() extends IO() {
	var x : Nothing = abort("");
}
