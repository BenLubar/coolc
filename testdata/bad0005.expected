testdata/bad0005.cool:2:10: cannot declare attribute of type Nothing
